string userInput
int64 channel
int64 value
---
bool result
